Pages/Hotels/ProjectPage.js

```diff
@@ -225,8 +232,10 @@ class ProjectPage{
         await browser.pause(2000)
```

Comment:
You using lot of browser.pause(), its not efficient. Use waitFor() or waitUntil().